xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -32,24 +37,20 @@ constexpr int ASS_BORDER_STYLE_SQUARE_BOX = 4; // Square box + outline
 
 constexpr int ASS_FONT_ENCODING_AUTO = -1;
 
-// Directory where user defined fonts are located (and where mkv fonts are extracted to)
+// Directory where user defined fonts are located (and where temporary MKV fonts are extracted to)
```

Comment:
```suggestion
// Directory where user defined fonts are located (and where MKV fonts are extracted and temporary stored)
```
i.e. the fonts themselves are not temporary :)